roles/container_engine/rkt/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Copy rkt binary from downloaddir
+  command: rsync -piu "{{ rkt_bin_dir }}/rkt" "{{ bin_dir }}/rkt"
+  changed_when: false
+
+- name: Create rkt network config directory
+  file:
+    path: "{{ rkt_netconfig_dir }}"
+    state: directory
+    owner: root
+    recurse: yes
+
+- name: Configure flannel network plugin for rkt
```

Comment:
No this is not required, it's an option. rkt can work with its own networking feature.
I'll work on the network plugins options when i'll get a first working version with rkt.